scripts/cat/cats.py

```diff
@@ -458,33 +448,33 @@ def init_generate_cat(self, skill_dict):
         )
 
         # Personality
-        self.personality = Personality(kit_trait=self.is_baby())
+        self.personality = Personality(kit_trait=self.age.is_baby())
 
         # experience and current patrol status
-        if self.age in ["young", "newborn"]:
+        if self.age in [CatAgeEnum.YOUNG_ADULT, CatAgeEnum.NEWBORN]:
```

Comment:
This doesn't mean the same thing. `young` is a kitten, not a young adult /info
Though this does make me think this section of code might need looking at more fully to see where we're coming from, as a cat's age should never have *been* "young"...
```suggestion
        if self.age.is_baby():
```